src/settings.py

```diff
@@ -325,6 +326,7 @@
 PING_IF_NUMS_ACCS = {}
 
 is_role = lambda plyr, rol: rol in ROLES and plyr in ROLES[rol]
+start_votes_required = lambda num_players: min((num_players + 1) / 2.2, 4)
```

Comment:
The other lambdas in settings.py shouldn't be in settings.py because they aren't settings. Legacy code doesn't give an excuse for new code to do dumb things, however. This lambda should instead be hardcoded into the function that actually runs it (or if more than one function runs it, then move it into wolfgame.py as-is so we don't needlessly duplicate code).

In terms of making it configurable, you'd use two variables -- var.START_VOTES_SCALE and var.START_VOTES_MAX. I'd also reconfigure the algorithm to act more like gunner bullet determination, so `min(ceil(num_players * var.START_VOTES_SCALE), var.START_VOTES_MAX)`. Doing it that way allows flexibility in the scaling as well as easily disabling it entirely if someone doesn't want it. Default values would be var.START_VOTES_SCALE = 0.4 and var.START_VOTES_MAX = 4, which retains the table you specified above.